website/language.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 """Various text used throughout the website, e.g. status messages, errors, etc.
 """
-
+from django.utils.translation import ugettext_lazy as _
```

Comment:
This was a request from me. We can move the variable back to the API side, I'm fine with that. There was some evolution here, and this might have evolved the wrong way. 

@pattisdr, let's go ahead and move the text into a place in the API that is visible for whatever needs to see this.